templates/scenarios/sso-tab-js/templates/azure/main.bicep

```diff
@@ -0,0 +1,18 @@
+@secure()
+param provisionParameters object
```

Comment:
Is this variable defined repeatedly? I guess you want to put all the variable definitions in the` config.bicep` file, I don't know if I understand it correctly.

Reply:
I just copy the biceps from current teamsfx project. Here the config.bicep is useless, but I don't want to make the change in this PR.